From cb8fedfd8276a332102fec61b406bec15c305b13 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Mon, 9 May 2016 16:12:40 +0800
Subject: [PATCH] ARM64: dts: rockchip: add ctrl-base for rk3399

Add ctrl-base for rk3399 to make emmc-phy work.

Change-Id: Iffb7a6dbe0b17d27c2cca4b2b99ddbc4e0736f15
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 70492a67d891..1df227179b6c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -311,6 +311,7 @@
 		reg-offset = <0xf780>;
 		#phy-cells = <0>;
 		rockchip,grf = <&grf>;
+		ctrl-base = <0xfe330000>;
 		status = "disabled";
 	};
 
-- 
2.35.3

